draft-ietf-teep-architecture.md

```diff
@@ -1006,8 +1006,72 @@ applications to interact with an agent for the TAM and TEE
 interaction.
 
 # Attestation
-
-## Attestation Hierarchy
+Attestation is the process through which one entity (an attestor) presents a series of
+claims to another entity (a verifier), and provides sufficient proof that the claims
+are true. Different verifiers may have different standards for attestation proofs
+and not all attestations are acceptable to every verifier. TEEP attestations are based
+upon the use of an asymmetric key pair under the control of the TEE to create digital
+signatures across a well-defined claim set.
+
+In TEEP, the primary purpose of an attestation is to allow a device to prove to TAMs 
+and SPs that a TEE in the device has particular properities, was built by a particular 
+manufacturer, or is executing a particular TA. Other claims are possible; this architecture 
+specification does not limit the attestation claims, but defines a minimal set of claims 
+required for TEEP to operate properly. Extensions to these claims are possible, but are not
+defined in the TEEP specifications. Other standards or groups may define the format and semantics
+of extended claims. The TEEP specification defines the claims format such that these extended claims
+may be easily included in a TEEP attestation message. 
+
+As of the writing of this specification, device and TEE attestations have not been standardized 
+across the market. Different devices, manufacturers, and TEEs support different attestation
+algorithms and mechanisms. In order for TEEP to be inclusive, the attestation format shall
+allow for both proprietary attestation signatures, as well as a standardized form of attestation
+signature. Either form of attesation signature may be applied to a set of TEEP claims, and 
+both forms of attestation shall be considered conformant with TEEP. However, it should be recognized 
+that not all TAMs or SPs may be able to process all proprietary forms of attestations. All TAMs
```

Comment:
Should we at least say that if they recognize the format but not certain claims that they should ignore unrecognized claims?